app/scripts/modules/core/src/pipeline/config/triggers/cron/cronPicker.html

```diff
@@ -297,6 +298,7 @@
       </div>
       <div class="row slide-in" ng-if="validation.messages.error">
         <div class="col-md-12 error-message">
+          <p>This trigger will NEVER fire.</p>
```

Comment:
another option is to not let the user save the pipeline... thoughts?

Reply:
I think adding this message is enough.